tests/src/JIT/Regression/JitBlue/GitHub_12761/GitHub_12761.il

```diff
@@ -0,0 +1,165 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+.assembly extern mscorlib {}
+
+.assembly GitHub_12761
+{
+}
+
+// =============== CLASS MEMBERS DECLARATION ===================
+
+.class public auto ansi beforefieldinit Test
+       extends [mscorlib]System.Object
+{
+  .class sequential ansi sealed nested public beforefieldinit IntWrapper
+         extends [mscorlib]System.ValueType
+  {
+    .field public int32 i
+  } // end of class IntWrapper
+
+  // Sum1 and Sum2 are have tail-recursive calls that the jit turns into loops.
+  // Both methods access locals that are not initialized by msil instructions; instead,
+  // the methods have .locals init directives. The jit has to initialize the locals on each iteration
+  // of the loop. The first test has a scalar local, the second test has a struct local.
+
+  .method public hidebysig static int32  Main() cil managed
+  {
+    .entrypoint
+    .maxstack  8
+              ldc.i4.0
+              ldc.i4.s   10
+              call       int32 Test::Sum1(int32,
+                                          int32)
+              ldc.i4.s   55
+              beq      TEST2
+
+              ldc.i4.0
```

Comment:
I don't think that this is consistent across the tests, but the old tests usually returned -1 for failure, and 100 for pass. I like that a little better than returning 0 for failure, since that's often considered a success return value.